assignments/objects.js

```diff
@@ -11,54 +11,132 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+	id: 0,
+	name: "Example",
+	email: "examples@you.edu",
+	gender: "F"
+};
 
 // Write your intern objects here:
 
-
-// ==== Challenge 2: Reading Object Data ==== 
+const intern1 = {
+	id: 1,
+	name: "Mitzi",
+	email: "mmelloy0@psu.edu",
+	gender: "F"
+};
+
+const intern2 = {
+	id: 2,
+	name: "Kennan",
+	email: "kdiben1@tinypic.com",
+	gender: "M"
+};
+
+const intern3 = {
+	id: 3,
+	name: "Keven",
+	email: "kmummery2@wikimedia.org",
+	gender: "M"
+};
+
+const intern4 = {
+	id: 4,
+	name: "Gannie",
+	email: "gmartinson3@illinois.edu",
+	gender: "M"
+};
+
+const intern5 = {
+	id: 5,
+	name: "Antonietta",
+	email: "adaine5@samsung.com",
+	gender: "F"
+};
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(intern1.name);
 // Kennan's ID
-
+console.log(intern2.id);
 // Keven's email
-
+console.log(intern3.email);
 // Gannie's name
-
+console.log(intern4.name);
 // Antonietta's Gender
+console.log(intern5.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+const kennan = intern2;
+
+kennan.speak = function () {
+	return `Hello my name is ${this.name}`;
+};
+
+console.log(kennan.speak());
 
```

Comment:
This was an interesting way to give intern2/kennan a speak function. If you do a console.log(kennan) _or_ console.log(intern2), it will show that function is now part of the intern2/kennan object. I thought that was pretty cool.

{id: 2, name: "Kennan", email: "kdiben1@tinypic.com", gender: "M", speak: ƒ}